### ISSUE TITLE
Splunk node - Unable to create a user

### ISSUE THREAD

**Issue by Analystn8n** [n8n-io/n8n#2348]
**Describe the bug**
When using the Splunk node in order to create a new user, the field "roles" trigger an error:
```
Error: Non-whitespace before first tag.
Line: 0
Column: 1
Char: [
    at error (/usr/local/lib/node_modules/n8n/node_modules/sax/lib/sax.js:651:10)
    at strictFail (/usr/local/lib/node_modules/n8n/node_modules/sax/lib/sax.js:677:7)
    at beginWhiteSpace (/usr/local/lib/node_modules/n8n/node_modules/sax/lib/sax.js:951:7)
    at SAXParser.write (/usr/local/lib/node_modules/n8n/node_modules/sax/lib/sax.js:1006:11)
    at Parser.exports.Parser.Parser.parseString (/usr/local/lib/node_modules/n8n/node_modules/xml2js/lib/parser.js:323:31)
    at Parser.parseString (/usr/local/lib/node_modules/n8n/node_modules/xml2js/lib/parser.js:5:59)
    at Object.exports.parseString (/usr/local/lib/node_modules/n8n/node_modules/xml2js/lib/parser.js:369:19)
    at /usr/local/lib/node_modules/n8n/node_modules/n8n-nodes-base/dist/nodes/Splunk/GenericFunctions.js:57:18
    at new Promise (<anonymous>)
    at parseXml (/usr/local/lib/node_modules/n8n/node_modules/n8n-nodes-base/dist/nodes/Splunk/GenericFunctions.js:56:12)
```
This error is also trigger when updating a user.

**To Reproduce**
1. Add Splunk node
2. Add Splunk API creds
3. Select Resource : User
4. Select Operation : Create
5. Enter the parameters Name, Roles and Password

**Expected behavior**
Create the user or update its role(s).

**Environment (please complete the following information):**
 - n8n Version [0.142.0]
 - official docker image n8nio/n8n:0.142.0

**Comment by janober** [comment]
Thanks a lot for reporting this issue @Analystn8n. The problem got fixed and will be released with the next version.

**Comment by janober** [comment]
Got released with n8n@0.145.0
